osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/smartmetering/ClockStatusBit.java

```diff
@@ -45,6 +48,10 @@ public String toString() {
         return forClockStatus(clockStatus & 0xFF);
     }
 
+    // here returning null is independed of the actual ClockStatusBit that is later used to make the list. This meand
+    // that returning an empty list would falsly suggest that there is an empty list, where in fact there is no valid
+    // list to return
+    @SuppressWarnings("squid:S1168")
```

Comment:
I don't think we should suppress this warning, and return an empty set.
Write unit tests for all the code calling this method to make sure everything keeps on working ok.